src/utils/index.js

```diff
@@ -986,6 +986,40 @@ export const transformStateDataForImport = state => {
     sanitizedState = { ...sanitizedState, stageFocus: STANDARD_VIEW_LIST[0] }
   }
 
+  // NOTE: This is a mitigation for
+  // https://github.com/jeremyckahn/farmhand/issues/546. There's no expected
+  // scenario where a cow would be present in cowBreedingPen but not
+  // cowInventory, but at least one player's game somehow got into that state.
+  // This block detects such an invalid state and corrects it.
+  {
+    const { cowId1, cowId2 } = sanitizedState.cowBreedingPen
+
+    const cowPenIdMap = sanitizedState.cowInventory.reduce(
```

Comment:
i know i already submitted approval but as soon as i did it occurred to me this might actually be a little simpler logic wise to use .some like 'const isCowMissing = [cow1id, cow2id].some(// check cow
pen for existence)' then your later if reads like 'if (isCowMissing) reset breeding pen'

sorry this comment is horribly formatted i'm on mobile. the way you have it works too i just wanted to offer a suggestion

Reply:
I like it! This is done in https://github.com/jeremyckahn/farmhand/commit/b6484cc7a8b08911da0e19d1fbe1ab125e267e34.